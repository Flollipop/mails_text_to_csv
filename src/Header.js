import React from 'react';

const headerLabel = 'Email address parser to csv';

export function Header() {
  return <header className='App-header'>{headerLabel}</header>;
}
