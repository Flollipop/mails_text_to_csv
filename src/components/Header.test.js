import React from 'react';
import ReactDOM from 'react-dom';
import { Header } from './Header';
import renderer from 'react-test-renderer';

test('render header - smoke test', () => {
  const div = document.createElement('div');
  ReactDOM.render(<Header />, div);
  ReactDOM.unmountComponentAtNode(div);
});

test('Header snapshot', () => {
  const tree = renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
});
