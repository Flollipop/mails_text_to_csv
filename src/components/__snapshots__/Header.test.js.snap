// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header snapshot 1`] = `
<header
  className="App-header"
>
  Email address parser to csv
</header>
`;
